include/hello_log.hrl

```diff
@@ -2,53 +2,73 @@
+               ?PREP_SC(LogId, "Hello handler with callback '~p' and service id '~p' answered async request."),
+               [CallbackModule, HandlerId])).
+
+-define(LOG_REQUEST_request(CallbackModule, HandlerId, Request, Response, Time, LogId),
+    lager:debug(?REQ_TRACES(CallbackModule, HandlerId, Request, ok, Response, LogId),
+               ?PREP_SC(LogId, "Hello handler with callback '~p' and service id '~p' answered synced request in '~w' ms."),
+               [CallbackModule, HandlerId, Time])).
+
+-define(LOG_REQUEST_request_stop(CallbackModule, HandlerId, Request, Response, Reason, Time, LogId),
+    lager:info(lists:append(?REQ_TRACES(CallbackModule, HandlerId, Request, ok, Response, LogId), [{hello_error_reason, Reason}]),
+               ?PREP_SC(LogId, "Hello handler with callback '~p' and service id '~p' answered request and stopped with reason ~w in ~w ms."),
+               [CallbackModule, HandlerId, Reason, Time])).
+
+-define(LOG_REQUEST_request_no_reply(CallbackModule, HandlerId, Request, Time, LogId),
+    lager:debug(?REQ_TRACES(CallbackModule, HandlerId, Request, error, LogId),
+               ?PREP_SC(LogId, "Hello handler with callback '~p' and service id '~p' failed to answer request in ~w ms."),
+               [CallbackModule, HandlerId, Time])).
+
+-define(LOG_REQUEST_request_stop_no_reply(CallbackModule, HandlerId, Request, Time, LogId),
+    lager:info(?REQ_TRACES(CallbackModule, HandlerId, Request, error, LogId),
+               ?PREP_SC(LogId, "Hello Handler with callback '~p' and service id '~p' failed to answer request and stopped with reason normal in ~w ms."),
+               [CallbackModule, HandlerId, Time])).
+
+-define(LOG_REQUEST_request_stop_no_reply(CallbackModule, HandlerId, Request, Reason, Time, LogId),
+    lager:info(lists:append(?REQ_TRACES(CallbackModule, HandlerId, Request, error, LogId), [{hello_error_reason, Reason}]),
+               ?PREP_SC(LogId, "Hello handler with callback '~p' and service id '~p' failed to answer request and stopped with reason ~w in ~w ms."),
+               [CallbackModule, HandlerId, Reason, Time])).
+
+-define(LOG_REQUEST_bad_request(CallbackModule, HandlerId, Request, Reason, LogId),
+    lager:info(lists:append(?REQ_TRACES(CallbackModule, HandlerId, Request, error, LogId), [{hello_error_reason, Reason}]),
+                ?PREP_SC(LogId, "Hello handler with callback '~p' and service id '~p' dismissed bad request."),
+                [CallbackModule, HandlerId])).
+
+-define(LOG_WARNING_reason(CallbackModule, HandlerId, Msg, Args, Reason, LogId),
+    lager:info(?DEFAULT_META( [ {hello_handler_callback, CallbackModule}, {hello_error_reason, Reason},
+                                {hello_service_id, HandlerId}],
+                                LogId)), Msg, Args).
+
+%% Generic
+-define(PREP_SC(LogId, Msg), lists:append([integer_to_list(element(2, LogId)), " - ", Msg])).
```

Comment:
@ContentSafe status codes in HTTP are for parsers, people prefer readable text. My point is if we have it in meta maybe we could get rid of it from text messages?